sigma_/sigma.py

```diff
@@ -34,10 +37,22 @@ def __init__(self, event_data: Dict) -> None:
             if k in ('Channel', 'EventID'):
                 json_body[k] = v
         body = {k: v for k, v in json_body.items() if v}
+        tags = {}
+        commandline_keys = ["CommandLine", "ParentCommandLine"]
+        if any(k in body for k in commandline_keys):
+            for item in commandline_keys:
+                v = body.get(item)
+                if v:
+                    uris = set(findall(uri_pattern, v.encode()))
+                    if uris:
+                        if not tags:
```

Comment:
nitpick: tags could be a defaultdict(list) instance. Would eliminate the need for the check and you would just extend if uris